### ISSUE TITLE
Remove Translate requirement

### ISSUE THREAD

**Issue by vojtasvoboda** [jawainc/simplecontact#36]
This plugin can be used also at sites without more localisations, so plugin should not depend on Translate, am I right?

Maybe you can define function `trans` somehow like that?

```
if (!function_exists('trans')) {
    function trans($s) {
        return $s;
    }
}
```

**Comment by jawainc** [comment]
Many users have difficulty in translating, so i decide to use translate dependency

**Comment by vojtasvoboda** [comment]
Yes translating is great idea. But what if I have a website only with one locale? I don't want to download another plugin for Translations.

**Comment by jawainc** [comment]
People have contributed for languages, so they want it to be more global not just targeting one language. right ?

**Comment by vojtasvoboda** [comment]
Yes I know. Translations are great. But your plugin CAN work without Translate plugin. Just try it. I've tried it now and it works great.

Just remove this line: https://github.com/jawainc/simplecontact/blob/master/Plugin.php#L22

So peoples with one-language sites don't have to download Translate plugin for nothing.

**Comment by jawainc** [comment]
ok, i'll do it on coming weekend
